src/utils/Cortex.js

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Missing Copyright header

Reply:
There should be a linting rule for this :)

Reply:
FYI @epmkelly